controllers/admin/AdminCarrierWizardController.php

```diff
@@ -266,12 +266,12 @@ public function renderStepThree($carrier)
                             [
                                 'id' => 'shipping_handling_on',
                                 'value' => 1,
-                                'label' => $this->trans('Enabled', [], 'Admin.Global'),
+                                'label' => $this->trans('Yes', [], 'Admin.Global'),
```

Comment:
Why changing these translations? :thinking:

Reply:
@PierreRambaud To make this consistent with the other pages avoid : 

![image](https://user-images.githubusercontent.com/16455155/93055169-cb89e680-f66a-11ea-9a5f-5c971f932623.png)

Reply:
@LouiseBonnard @Junebyun could you help us ?
I think we need to harmonize the labels for switch inputs. Either we use "Yes" and "No" or "Enabled" and "Disabled".
WYDF ?

@PierreRambaud  FYI, The current behavior is "Yes" or "No" in all cases. Currently Labels in forms are not taken into account.
See https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/default/template/helpers/form/form.tpl#L423
https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/default/template/helpers/form/form.tpl#L425

Reply:
Hi @PululuK, that's an interesting question for @Julievrz, our new Product Content Manager! And once the decision made, it might be great to relate it to https://github.com/PrestaShop/PrestaShop/issues/19142. :-)

Reply:
@LouiseBonnard  @PierreRambaud  news ?

Reply:
@PululuK I'm not the one who decides about wordings :(

Reply:
@LouiseBonnard @PierreRambaud But I don't understand why this PR is waiting for wording ? The current behavior is "Yes" or "No".

See https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/default/template/helpers/form/form.tpl#L423 and https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/default/template/helpers/form/form.tpl#L425

If you want to modify wordings this is not in the scope of my PR. 

See https://github.com/PrestaShop/PrestaShop/pull/20653#discussion_r487705269

Reply:
Yes/No is ok for me. @PululuK @PierreRambaud

Reply:
Thank you @Julievrz  !

Reply:
@Julievrz / @LouiseBonnard If the wording is OK : can you put the label "Wording :heavy_check_mark:" ?